src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -59,7 +59,25 @@ def _checkXRDUtilsExist(self):
 
         return foundXrdcp & foundXrdfs
 
-    def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=None):
+    def getAuthEnv(self, auth_method=None, force_method=False):
+        """
+        Get environment variables for stageout command based on the selected authentication method.
+        """
+        authEnv = ""
+        if auth_method == "X509":
```

Comment:
Hi @khurtado , should we store `BEARER_TOKEN`, `BEARER_TOKEN_FILE`, and `X509_USER_PROXY` in backup variables before unsetting them?

Reply:
I don't think it is needed because we are not exporting the variables in the shell environment, but rather doing a local modification via `env ... <cmd>`.